tools/distpartitioning/utils.py

```diff
@@ -108,24 +128,35 @@ def get_edge_types(schema):
 
     return etypes_map, etypes
 
-def get_ntypes_map(schema): 
+def get_ntypes_map(node_tids): 
```

Comment:
this function name is confusing. this is actual to get the ID range for each node type.

Reply:
changed the function name to "get_gnid_range_map" which is appropriate in this case. Here we take the node's tid ranges list and return their corresponding gnid range list.